lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journeys/app-doc-check.yaml

```diff
@@ -36,19 +36,19 @@ nestedJourneyStates:
       next:
         exitEventToEmit: next
       access-denied:
-        targetState: RESET_SESSION_INCOMPLETE_DL
+        targetState: REMOVE_DCMAW_FROM_SESSION
       alternate-doc-invalid-dl:
         targetState: RESET_SESSION_INVALID_DL
 
-  RESET_SESSION_INCOMPLETE_DL:
+  REMOVE_DCMAW_FROM_SESSION:
     response:
       type: process
       lambda: reset-session-identity
       lambdaInput:
-        resetType: ALL
+        resetType: DCMAW
```

Comment:
Do you think the other reset should be DCMAW as well? I don't think there are currently any scenarios where it matters, but it feels more consistent.

(The D02 only really invalidates the DCMAW VC)